ingest/expression_files/expression_files.py

```diff
@@ -102,4 +113,46 @@ def set_data_array_gene_expression_values(
             linear_data_id,
             self.study_id,
             self.study_file_id,
-        ).get_data_array()
+        ).get_data_array():
+            yield model
+
+    def load_expression_file(self, gene_docs: List, data_array_documents: List):
+        """
+        Load gene and data_array models into mongoDB
+        """
+        gene_doc_bulk_write_results = None
+        data_array_bulk_write_results = None
+        start_time = datetime.datetime.now()
+
+        # Creating Mongo bulk operations
+        data_array_bulk_operations = list(
+            map(lambda model: InsertOne(model), data_array_documents))
```

Comment:
Is there a reason why we're doing a `list` of `InsertOne` operations and calling `bulk_write()`?  We're not doing a mix of write/update/delete operations, so it would stand to reason that calling `insert_many()` would suffice once the list of documents to insert reaches a certain pre-set limit.

Reply:
I don't think so. Updated in 5441f62